homeassistant/components/light/__init__.py

```diff
@@ -282,7 +282,14 @@ class SetIntentHandler(intent.IntentHandler):
                 pars = params.copy()
                 pars[ATTR_PROFILE] = Profiles.get_default(light.entity_id)
                 preprocess_turn_on_alternatives(pars)
-            await light.async_turn_on(**pars)
+            if ATTR_BRIGHTNESS in pars and pars[ATTR_BRIGHTNESS] == 0:
+                # Zero brightness: Turn the light off
+                for k in list(pars.keys()):
```

Comment:
pars = {k: v for k,v in pars.items() if k not in [ATTR_TRANSITION, ATTR_FLASH]}

Reply:
Minor typo: `…k in [ATTR…`

Reply:
Fixed!